exec/jdbc/src/test/java/org/apache/drill/jdbc/test/Drill2489CallsAfterCloseThrowExceptionsTest.java

```diff
@@ -537,7 +547,15 @@ public void testclosedPreparedStmtOfOpenConnMethodsThrowRight() {
         new ClosedPreparedStatementChecker(PreparedStatement.class,
                                            closedPreparedStmtOfOpenConn);
 
-    checker.testAllMethods();
+    //List of methods now supported
+    @SuppressWarnings("serial")
```

Comment:
Please revert these changes in this class once calls to Avatica methods are used.

Reply:
Not sure what you meant by the change. Should I remove the SuppressWarnings?